### PR TITLE
[DDCE-818]:§8 P2 - What is the other official’s name?

### PR DESCRIPTION
**Pull request by anburoach**
# DDCE-818

# New feature

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

# PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

# Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

# Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by abhijeetardale** [comment]
please update and rebase ATs

File: conf/messages.cy [hunk 1]
```diff
@@ -737,6 +737,22 @@ charityOtherOfficials.p2 = (W)Other officials are people in the charity who can
 charityOtherOfficials.p3 = (W)They will normally be the trustees or directors of the charity (if the charity is a company), or community amateur sports club (CASC) officials.(W)
 charityOtherOfficials.p4 = (W)Adding information about other officials takes around 5 minutes.(W)
 
+# Authorised Officials Name Messages
```
Comment (abhijeetardale) [inline conf/messages.cy:740]: update comment for Other

File: conf/messages.en [hunk 2]
```diff
@@ -737,6 +737,22 @@ charityOtherOfficials.p2 = Other officials are people in the charity who can tel
 charityOtherOfficials.p3 = They will normally be the trustees or directors of the charity (if the charity is a company), or community amateur sports club (CASC) officials.
 charityOtherOfficials.p4 = Adding information about other officials takes around 5 minutes.
 
+# Authorised Officials Name Messages
```
Comment (abhijeetardale) [inline conf/messages.en:740]: update comment for Other

**Comment by anburoach** [comment]
Rebased and finished review comments

**Pull request merged**